include/rocksdb/utilities/backupable_db.h

```diff
@@ -122,16 +131,19 @@ struct BackupableDBOptions {
   // Default: INT_MAX
   int max_valid_backups_to_open;
 
-  // If true, backup SST filenames consist of file_number, crc32c, db_session_id
-  // if false, backup SST filenames consist of file_number, crc32c, file_size
+  // Naming option for share_files_with_checksum table files
+  // If it is set to kChecksumAndDbSessionId,
```

Comment:
I think these details should be comments on the enumerators.

We should also mention (again) that the naming scheme is not an issue for version upgrade or backup directory compatibility (for reasons I mentioned above).